src/options.rs

```diff
@@ -118,6 +120,23 @@ impl SftpOptions {
         self.read_end_buffer_size
             .unwrap_or_else(|| NonZeroUsize::new(1024).unwrap())
     }
+
+    /// Set the write buffer limit for tokio compat file.
+    /// If the write buffer size is larger than the limit in tokio compat file,
+    /// then it will flush one write buffer and following async write operation will be pending.
```

Comment:
```suggestion
    /// If [`crate::file::TokioCompatFile`] has hit the write buffer limit
    /// set here, then it will flush one write buffer and continue
    /// sending (part of) the buffer to the server, which could be buffered.
```